shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/metadata/persist/service/impl/SchemaRulePersistServiceTest.java

```diff
@@ -56,6 +56,15 @@ public void assertLoadWithExistedNode() {
         assertThat(actual.size(), is(1));
     }
     
+    @Test
+    public void assertIsExisted() {
+        when(repository.get("/metadata/foo_db/foo_db/active_version")).thenReturn("0");
+        when(repository.get("/metadata/foo_db/foo_db/versions/0/rules")).thenReturn(readYAML());
+        SchemaRulePersistService schemaRulePersistService = new SchemaRulePersistService(repository);
+        assertThat(schemaRulePersistService.isExisted("foo_db"), is(true));
+        assertThat(schemaRulePersistService.isExisted("foo_db_1"), is(false));
```

Comment:
Would it be more reasonable to use `assertTrue` and `assertFalse`.

Reply:
@yx9o Yes, I see what you mean